docs/proposals/accepted/observatorium.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "2021-05: Observatorium"
+menu:
+  docs:
+    parent: "acc-proposals"
+toc: true
+---
+
+Taken from [Open Source Design Doc Template](https://docs.google.com/document/d/1zeElxolajNyGUB8J6aDXwxngHynh4iOuEzy3ylLc72U/edit#).
+
+## Observatorium
+
+Initial Observatorium Design Proposal
+
+* **Owners:**
+  * `@bwplotka`
```

Comment:
I don't think this is appropriate for the proposal for "observatorium.md" as it somewhat implies the idea for all of Observatorium is entirely owned by you. Maybe you meant to be the owner for a refactor or of a new API? Otherwise should the whole team be here?

Reply:
Well.. it's really about ... proposal? ;p

Reply:
but ok, I am fine with removing myself, it's def not about me